src/main/java/au/org/democracydevelopers/raireservice/persistence/entity/Contest.java

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2024 Democracy Developers
+The Raire Service is designed to connect colorado-rla and its associated database to
+the raire assertion generation engine (https://github.com/DemocracyDevelopers/raire-java).
+
+This file is part of raire-service.
+raire-service is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+raire-service is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+You should have received a copy of the GNU Affero General Public License along with raire-service. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package au.org.democracydevelopers.raireservice.persistence.entity;
+
+import jakarta.persistence.*;
+import org.springframework.data.annotation.ReadOnlyProperty;
+
+/*
+ * A contest class used for reading contest data out of the corla database.
+ * This class omits the fields that are not relevant to input validation - we only care about
+ * checking whether the request we received makes sense for IRV assertion generation or retrieval.
+ */
+@Entity
+@Table(name = "contest")
+public class Contest {
+
+  /**
+   * Contest ID.
+   */
+  @Id
+  @Column(nullable = false, updatable = false)
+  @ReadOnlyProperty
+  private long id;
+
+  /**
+   * The name of the contest.
+   */
+  @ReadOnlyProperty
+  @Column(name = "name", nullable = false, updatable = false)
+  private String name;
+
```

Comment:
Can you put the getters at the bottom? Ordering preference:

1. Class (static) variables: First the public class variables, then the protected, and then the private.

2. Instance variables: First public, then protected, and then private.
Constructors

3. Methods: These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.